test/namedtuple.jl

```diff
@@ -21,9 +21,23 @@
 @test (a=3,)[:a] == 3
 @test (x=4, y=5, z=6).y == 5
 @test (x=4, y=5, z=6).z == 6
+@test (x=4, y=5, z=6)[(:x, :y)] == (x=4, y=5)
+@test (x=4, y=5, z=6)[(:x,)] == (x=4,)
+# If the following are included, is it worth putting in 1 line with above, eg:
+# @test (x=4, y=5, z=6)[(:x, :y)] == (x=4, y=5, z=6)[[:x, :y]] == (x=4, y=5)
```

Comment:
I think it's fine to leave the extra line out; if you remove these lines, I think this is ready to merge